viam-cartographer/src/carto_facade/carto_facade.h

```diff
@@ -359,7 +356,7 @@ class CartoFacade {
     viam::carto_facade::config config;
     viam_carto_algo_config algo_config;
     std::string path_to_internal_state;
-    std::atomic<bool> started{false};
```

Comment:
I changed this atomic boolean to be an atomic enum to be able to better describe the different possible states of the CartoFacade.